modules/ROOT/pages/_partials/commons/common-logging.adoc

```diff
@@ -29,94 +29,178 @@ include::{root-partials}_block-abstract.adoc[]
 endif::[]
 // End -- Output an abstract unless this inclusion is being used as part of an encompassing page
 
+:logging-classes: {url-api-class-log}, {url-api-class-log-file} and {url-api-class-log-config}
+ifeval::["{param-platform}"=="{platform-net}"]
+:logging-classes: {url-api-class-logging}
+endif::[]
+
+ifdef::is-android[]
+.Constraints
+endif::[]
+
+[NOTE]
+--
+The retrieval of logs from the device is out of scope of this feature.
```

Comment:
Boy... *That* is a shame!  Esp on Android, it is pretty hard to do.  Unless customers can get logs to us, they are essentially useless.

It is exactly the crack through which things fall: we, often "log it", as a way of addressing an inconsistency or an edge case in our design.  If we then don't provide a way of getting to the logs, well...

Reply:
What is the process for this? Is it anything like the SG_Collect https://docs.couchbase.com/sync-gateway/current/sgcollect-info.html